cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,25 +1,38 @@
-describe("Buy a t-shirt", () => {
+import { MenuContentPage } from "../page/index"
+import { ShoppingCartPage } from "../page/index"
+import { ShippingCartPage } from "../page/index"
+import { LoginPage } from "../page/index"
+import { PaymentStep } from "../page/index"
+import { AdressStep } from "../page/index"
+import { ProductsListPage } from "../page/index"
+
+const menuContentPage = new MenuContentPage()
+const shoppingCartPage = new ShoppingCartPage()
+const shippingCartStep = new ShippingCartPage()
+const loginPage = new LoginPage()
+const paymentStep = new PaymentStep()
+const adressStep = new AdressStep()
+const productsListPage = new ProductsListPage()
 
-    it("then the t-shirt should be bought", () => {
-      cy.visit("http://automationpractice.com/")
-      cy.get("#block_top_menu > ul > li:nth-child(3) > a").click()
-      cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click()
-      cy.get("[style*='display: block;'] .button-container > a").click()
-      cy.get(".cart_navigation span").click()
+describe("Buy a t-shirt", () => {
 
-      cy.get("#email").type("aperdomobo@gmail.com")
-      cy.get("#passwd").type("WorkshopProtractor")
-      cy.get("#SubmitLogin").click()
-      cy.get(".cart_navigation > .button > span").click()
-      cy.get("#cgv").click()
-      cy.get(".button[name='processCarrier']").click()
-      cy.get(".bankwire").click()
-      cy.get(".button[type='submit']").contains("I confirm my order").click()
-      cy.get("#center_column > div > p > strong")
-      .should("have.text", "Your order on My Store is complete.")
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage()
+    menuContentPage.goToTShirtMenu()
+    menuContentPage.AddToCart()
+    productsListPage.checkoutItemsInTheCar()
+    shoppingCartPage.goToPayment()
+    loginPage.typeUser("aperdomobo@gmail.com")
+    loginPage.typePassword("WorkshopProtractor")
+    loginPage.submitLoginCredentials()
+    adressStep.confirmAdresses()
+    shippingCartStep.termsAndConditionsBox()
+    shippingCartStep.processCarrier()
+    paymentStep.selectPaymentMethod()
+    paymentStep.confirmTheOrder()
+    paymentStep.validateConfirmationPaymentMessage("Your order on My Store is complete.")
```

Comment:
yo can create a variable to store the expected message and initialize it on the arrange section

Reply:
Also made for email and password :)